zebra-consensus/src/parameters/network_upgrade.rs

```diff
@@ -0,0 +1,163 @@
+//! Network upgrade consensus parameters for Zcash.
+
+use NetworkUpgrade::*;
+
+use std::collections::{BTreeMap, HashMap};
+use std::ops::Bound::*;
+
+use zebra_chain::types::BlockHeight;
+use zebra_chain::{Network, Network::*};
+
+/// A Zcash network protocol upgrade.
+//
+// TODO: are new network upgrades a breaking change, or should we make this
```

Comment:
If the plan is to always make a major version bump on network upgrades then I feel like the answer is yes, this is a breaking change, if only because its free.

Reply:
I think we should treat a new network upgrade, new or modified activation height, and new or modified consensus branch id as a new major version.

In particular, testnet may change activation heights and branch ids after a rollback, without affecting mainnet.